addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.xml

```diff
@@ -40,11 +40,11 @@
                         </t>
                     </div>
                     <div class="o_ThreadNeedactionPreview_core d-flex align-items-baseline">
-                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': inlineLastNeedactionMessageAsOriginThreadBody.length === 0 }">
+                        <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_inlineText text-truncate" t-att-class="{ 'o-empty': threadNeedactionPreviewView.isEmpty }">
                             <t t-if="threadNeedactionPreviewView.messageAuthorPrefixView">
                                 <MessageAuthorPrefix localId="threadNeedactionPreviewView.messageAuthorPrefixView.localId"/>
                             </t>
-                            <t t-esc="inlineLastNeedactionMessageAsOriginThreadBody"/>
+                            <span class="o_ThreadNeedactionPreview_coreItem o_ThreadNeedactionPreview_messageBody" t-esc="threadNeedactionPreviewView.inlineLastNeedactionMessageAsOriginThreadBody"/>
```

Comment:
Parent element already has the class `o_ThreadNeedactionPreview_coreItem`, it is not correct to have it here as well.